cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -3348,9 +3348,9 @@ pub(crate) fn emit(
             sink.put1(*imm);
         }
 
-        Inst::XmmUninitializedValue { .. } => {
-            // This instruction format only exists to declare a register as a `def`; no code is
-            // emitted.
+        Inst::XmmUninitializedValue { .. } | Inst::GprUninitializedValue { .. } => {
+            // These instruction formats only exist to declare a register as a
+            // `def`; no code is emitted.
```

Comment:
I know pre-existing, but can we add a bit more here about how it's used (to ease concerns of the reader that this looks wildly unsafe)? Basically something like: "This is always immediately followed by an instruction, such as `xor reg, reg`, that *semantically* reads this undefined value but *arithmetically* produces the same result regardless of its value."